packages/babel/README.md

```diff
@@ -75,6 +75,21 @@ Type: `String | RegExp | Array[...String|RegExp]`<br>
 
 A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should operate on. When relying on Babel configuration files you cannot include files already excluded there.
 
+### `filter`
+
+Type: (id: unknown) => boolean<br>
```

Comment:
doesn't this always receive a `string`?

Reply:
Originally, the type should be ReturnType<typeof createFilter> right?. As per https://github.com/rollup/plugins/blob/master/packages/pluginutils/types/index.d.ts#L52